src/main/java/io/jenkins/pluginhealth/scoring/PluginHealthScoring.java

```diff
@@ -24,10 +24,14 @@
 
 package io.jenkins.pluginhealth.scoring;
 
+import io.jenkins.pluginhealth.scoring.config.GithubConfiguration;
+
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
+@EnableConfigurationProperties(value = GithubConfiguration.class)
```

Comment:
Why do we need to use this annotation and mention the GithubConfiguration.class?

I read about this annotation and as per my understanding, it's helpful in quickly registering a @ConfigurationProperties annotated class in Spring. And GithubConfiguration class does have the @ConfigurationProperties annotation. Then I read about the @ConfigurationProperties annotation and found out that it helps us to easily read values from the yaml file that you've created. And since the yaml file has so many values and we are only concerned with the github.oauth that's why you've mentioned prefix="github" so that's easier to pin down which exact field we need to read from the yaml file.

Is this correct?